examples/symfony/src/AppBundle/AppBundle.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace AppBundle;
+
+use OpenCensus\Trace\Exporter\StackdriverExporter;
```

Comment:
Do you need these imports?

Reply:
Nope, I'll remove. These were remnants from when I had the integrations directly in `AppBundle` instead of the `OpenCensusBundle`